cmd/ipfs-cluster-ctl/main.go

```diff
@@ -453,9 +453,18 @@ in the cluster and should be part of the list offered by "pin ls".
 An optional replication factor can be provided: -1 means "pin everywhere"
 and 0 means use cluster's default setting. Positive values indicate how many
 peers should pin this content.
+
+An optional allocations argument can be provided, allocations should be a
```

Comment:
This needs to be updated to explain that the given allocations will have priority over automatically determined ones, but replication factors are respected.